AppCenter/AppCenter/Internals/History/Device/MSDeviceTrackerPrivate.h

```diff
@@ -162,6 +162,13 @@ static NSString *const kMSPastDevicesKey = @"pastDevicesKey";
  */
 - (void)setWrapperSdk:(MSWrapperSdk *)wrapperSdk;
 
+/**
+ * Set country code to use when building device properties.
+ *
+ * @param countryCode The two-letter ISO country code. @see https://www.iso.org/obp/ui/#search for more information.
```

Comment:
Nitpick: move `@see` to the new line for better readability

Reply:
@Quan-Jin Please run `./clang-format-changed-files.sh` to format your code before submitting the PR. The script will take care of any styling issues.